src/src/css/main.css

```diff
@@ -6,150 +6,8 @@
 
 @layer base {
```

Comment:
Might be useful for us to set up a group meeting and detail out styling for the app.

Reply:
One approach is to try and make our extension follow Google style language so as to fit in with the rest of the UI elements in the Google Calendar

Reply:
I like this, will look into it more because I am not familiar.